File: Server/src/main/java/xyz/sleepylux/hextechfriends/models/DTO/server/JoinAcceptedTC.java
package xyz.sleepylux.hextechfriends.models.DTO.server;

public class JoinAcceptedTC {
    private String opCode = "joinAcceptedTC";
    private String ownerSummonerName;
    private int ownerIconId;

    public void setOwnerSummonerName(String ownerSummonerName) { this.ownerSummonerName = ownerSummonerName; }

    public void setOwnerIconId(int ownerIconId) { this.ownerIconId = ownerIconId; }
}

File: Client/HextechFriendsClient/Protocol/Server/JoinAcceptedTC.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HextechFriendsClient.Protocol.Server
{
    class JoinAcceptedTC
    {
        public string opCode { get; set; }
        public string ownerSummonerName { get; set; }
        public int ownerIconId { get; set; }
    }
}
